src/components/home/Testimonials.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Flex, Grid, Heading, Text } from "@radix-ui/themes";
+
+import { partnerTestimonials } from "@/data/home";
+
+const Testimonials = () => {
+  return (
+    <Box p="7">
```

Comment:
For screen readers:
inside the first box having a section with an aria-labelledby + id can help give a screen reader a landmark to jump to
e.g.
`<section aria-labelledby="testimonials-heading">`